### PR TITLE
add proper sfx env variables to deployment

### PR DESCRIPTION
**Pull request by slernersplunk**
the deployment spec of the service is missing the realm and token variables needed to work correctly...

I've added this as an example starting on 152:

        env:
        - name: SFX_REALM
          value: # set your realm here
        - name: SFX_TOKEN
          value: # set your token here

We'll need to change the referring docs as well indicating that these need to be set.
Taking from $() variables doesn't seem to want to work in this example.

**Description:** <Describe what has changed.>

**Link to tracking Issue:** <Issue number if applicable>

**Testing:** <Describe what testing was performed and which tests were added.>

**Documentation:** <Describe the documentation added.>

### CODE REVIEW

**Comment by bogdandrutu** [comment]
@slernersplunk please sign the CLA

**Comment by slernersplunk** [comment]
I'm waiting on CLA approval from @flands when I get authorized I can sign

**Review by jrcamp — APPROVED** [review]

**Comment by slernersplunk** [comment]
CLA signed

**Review by bogdandrutu — APPROVED** [review]

**Pull request merged**
